src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/TranscriptionArchiveRecordMapperImpl.java

```diff
@@ -159,7 +159,9 @@ private RecordMetadata createArchiveRecordMetadata(ExternalObjectDirectoryEntity
             .publisher(armDataManagementConfiguration.getPublisher())
             .recordClass(armDataManagementConfiguration.getTranscriptionRecordClass())
             .recordDate(formatDateTime(currentTimeHelper.currentOffsetDateTime()))
-            .eventDate(formatDateTime(transcriptionDocument.getUploadedDateTime()))
+            .eventDate(formatDateTime(nonNull(transcriptionDocument.getRetainUntilTs())
```

Comment:
can call to transcriptionDocument.getRetainUntilTs() twice be avoided here? also indentation looks a bit weird (even though checkstyle probably dosen't care)

Reply:
Done